core/templates/dev/head/player/exploration_player.html

```diff
@@ -105,6 +65,106 @@
     <feedback-popup></feedback-popup>
   </script>
 
+  <script type="text/ng-template" id="popover/informationCard">
+    <md-card class="md-padding">
+      <div ng-style="infoCardBackgroundImageCss" class="oppia-info-card-logo-thumbnail">
+        <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
+      </div>
+
+      <div class="oppia-info-card-content">
+        <p ng-if="explorationDescription">
+          <[explorationDescription | truncateAndCapitalize: 95]>
+        </p>
+        <p ng-if="!explorationDescription">No objective specified.</p>
+        <ul layout="row" class="card-metrics" layout-align="space-between center">
+          <li>
+            <span class="fa fa-star fa-lg"></span>
+            <[explorationRatings]>
+          </li>
+          <li>
+            <span class="fa fa-eye fa-lg"></span>
+            <[viewersCount | summarizeNonnegativeNumber]>
+          </li>
+          <li>
+            <span class="fa fa-clock-o fa-lg"></span>
+            <[getLocaleAbbreviatedDatetimeString(explorationLastUpdatedMsec)]>
+          </li>
+          <ul layout="row" layout-align="space-between center" class="oppia-info-card-exploration-contributors-profile">
+            <img src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
+            <li ng-repeat="ContributorUsername in explorationContributorUsernames| limitTo: 2"
+              tooltip="<[ContributorUsername]>" tooltip-placement="top">
+              <img class="img-circle" ng-src="/images/general/user_mint_48px.png" alt="">
```

Comment:
Can these be a little smaller (say 20px by 20px) while keeping the ic_group_black_48dp.png icon at the same size as the others (star, eye, etc.)? The user icons should feel subsidiary to the 'header' icon.